### ISSUE TITLE
Abandon sourceforge

### ISSUE THREAD

**Issue by kbilsted** [super-csv/super-csv#78]
Hi team

In the other issue we discussed sourceforge. There seems to be many downloads there - but I'm not sure if it really are inflated numbers - downloads by bots.

On the other hand, github can be confusing at times.. e.g. finding releases. How about we create a documentation page called "downloading supercsv" that links to the github releases page and explains what to click? or just link to the release page. Then I'll ckise the sourceforge page.

if any of you still have feelings for sourceforge and wants to keep uploading to it, then by all means let's continue - but some one has to volunteer for the job :)

**Comment by ZioberMichal** [comment]
Hi, we already have [downloading page](http://super-csv.github.io/super-csv/downloading.html) (it even points to page on GitHub where all releases are available). I guess that 99% of developers downloads libs from Maven repositories. In my opinion, we can skip Sourceforge without detriment to the community. It even will be much clear if releases will be available only on Maven repositories and platform where source code is maintained.

**Comment by jamesbassett** [comment]
I agree that the current 'downloading' page has more than enough info, and that it's safe to no longer upload releases to SourceForge.

Just in case anyone is tempted to delete the SF project just remember that:
* the old website is still hosted there (each page redirects to it's GH equivalent) so links all over the net aren't suddenly broken
* the donation URL still goes there (the maven site plugin had issues with URLs with query params, so we let SF do the redirect).

**Comment by kbilsted** [comment]
@jamesbassett  the donations have equaled $10 in total. So not really a problem :)

**Comment by jamesbassett** [comment]
And that was probably one person! We certainly don't do open source for the money :)

**Comment by kbilsted** [comment]
@jamesbassett  I do... and Im poor ;-P
